### DISCUSSION TITLE
Iam User permission

### DISCUSSION THREAD

**Discussion by prian26** [trendmicro/cloudone-filestorage-plugins#74] [Q&A] [answered]
Hi All,

Im looking to configure cloud one for the first time.
1) Where in the template.yml file does it configure the iAM user?
2) How can i set PermissionBoundary for roles when creating the iAM user?
thanks,
Pv

**Comment by trend-jack-c-tang** [comment, marked as answer]
Hi @prian26,
File Storage Security (FSS) CloudFormation templates create IAM roles but not IAM users. To apply permissions boundary for those roles, you can specify `PermissionsBoundary` parameter when deploying FSS stacks. All deployment parameters and their description can be found here: https://cloudone.trendmicro.com/docs/file-storage-security/gs-deploy-all-in-one-stack/

Welcome to check out our getting started guide to configure Cloud One File Storage Security: https://cloudone.trendmicro.com/docs/file-storage-security/gs-before/
